Dockerfile

```diff
@@ -87,6 +87,9 @@ RUN set -x && \
     mv _output/pmem-csi-driver${BIN_SUFFIX} /usr/local/bin/pmem-csi-driver && \
     mv _output/pmem-vgm${BIN_SUFFIX} /usr/local/bin/pmem-vgm && \
     mv _output/pmem-ns-init${BIN_SUFFIX} /usr/local/bin/pmem-ns-init && \
+    if [ "$BIN_SUFFIX" = "-test" ]; then GOOS=linux GO111MODULE=on \
+        go build -o /usr/local/bin/pmem-dax-check ./test/cmd/pmem-dax-check && \
+	swupd bundle-add fio; fi && \
```

Comment:
Installing fio here doesn't actually work: you are adding it to the "binaries" build image, but where we want it is in the "pmem" image.

That image is derived from the "runtime" image where we already add bundles. You can add it there with:
```
RUN swupd update ${SWUPD_UPDATE_ARG} && swupd bundle-add file xfsprogs storage-utils $(if [ "$BIN_SUFFIX" = "-test" ]; then echo fio; fi) && rm -rf /var/lib/swupd /var/tmp/swupd
```